src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/global.yml

```diff
@@ -229,9 +229,6 @@ Resources:
             Resource: 
               - "*"
           - Effect: Allow
-            Condition:
```

Comment:
I think these got moved in the process of merging changes.

The condition should have been applied on the "sts:AssumeRole" action listed above it.
Could you update the PR to move these lines to be part of that statement?